Formula/the_platinum_searcher.rb

```diff
@@ -15,59 +16,15 @@ class ThePlatinumSearcher < Formula
   end
 
   depends_on "go" => :build
-
-  go_resource "github.com/BurntSushi/toml" do
-    url "https://github.com/BurntSushi/toml.git",
-    :revision => "f0aeabca5a127c4078abb8c8d64298b147264b55"
-  end
-
-  go_resource "github.com/jessevdk/go-flags" do
-    url "https://github.com/jessevdk/go-flags.git",
-    :revision => "b9b882a3990882b05e02765f5df2cd3ad02874ee"
-  end
-
-  go_resource "github.com/monochromegane/conflag" do
-    url "https://github.com/monochromegane/conflag.git",
-    :revision => "6d68c9aa4183844ddc1655481798fe4d90d483e9"
-  end
-
-  go_resource "github.com/monochromegane/go-gitignore" do
-    url "https://github.com/monochromegane/go-gitignore.git",
-    :revision => "38717d0a108ca0e5af632cd6845ca77d45b50729"
-  end
-
-  go_resource "github.com/monochromegane/go-home" do
-    url "https://github.com/monochromegane/go-home.git",
-    :revision => "25d9dda593924a11ea52e4ffbc8abdb0dbe96401"
-  end
-
-  go_resource "github.com/monochromegane/terminal" do
-    url "https://github.com/monochromegane/terminal.git",
-    :revision => "2da212063ce19aed90ee5bbb00ad1ad7393d7f48"
-  end
-
-  go_resource "github.com/shiena/ansicolor" do
-    url "https://github.com/shiena/ansicolor.git",
-    :revision => "a422bbe96644373c5753384a59d678f7d261ff10"
-  end
-
-  go_resource "golang.org/x/text" do
-    url "https://go.googlesource.com/text.git",
-    :revision => "da88cb54a2fe4ec041612bd04b321d6a79d3ea81"
-  end
-
-  go_resource "gopkg.in/yaml.v2" do
-    url "https://gopkg.in/yaml.v2.git",
-    :revision => "a83829b6f1293c91addabc89d0571c246397bbf4"
-  end
+  depends_on "godep" => :build
 
   def install
-    mkdir_p buildpath/"src/github.com/monochromegane"
-    ln_s buildpath, buildpath/"src/github.com/monochromegane/the_platinum_searcher"
-
     ENV["GOPATH"] = buildpath
-    Language::Go.stage_deps resources, buildpath/"src"
-
+    dir = buildpath/"src/github.com/monochromegane/the_platinum_searcher"
+    dir.install Dir["*"]
+    ln_s buildpath/"src", dir
+    cd dir
```

Comment:
Would be nice to make this `cd dir do` to match most usage elsewhere. We try to avoid vanilla `cd`s unless using them takes up a ridiculous amount of space or is problematic in some other way.